recipes-kernel/linux-firmware-rpidistro/linux-firmware-rpidistro_git.bb

```diff
@@ -47,19 +47,20 @@ do_install() {
 
     # add compat links. Fixes errors like
     # brcmfmac mmc1:0001:1: Direct firmware load for brcm/brcmfmac43455-sdio.raspberrypi,4-model-compute-module.txt failed with error -2
-    ln -s brcmfmac43455-sdio.txt ${D}${nonarch_base_libdir}/firmware/brcm/brcmfmac43455-sdio.raspberrypi,4-compute-module.txt
+    #ln -s brcmfmac43455-sdio.txt ${D}${nonarch_base_libdir}/firmware/brcm/brcmfmac43455-sdio.raspberrypi,4-compute-module.txt
```

Comment:
What's up with these comments?

Reply:
These symlinks are already part of new prebuilt. I have kept them in comments for reference.

Reply:
That sounds great! In that case I would propose to drop these from the recipe and reference them in it log. In this way the info remains easy to access while we don't have to maintain comments and references in the recipe. What do you think?

Reply:
done

Reply:
I hate to be nit picker but we missed a comment in git log for this drop.

Reply:
indeed I did. Fixed now.